jest.config.js

```diff
@@ -22,8 +22,9 @@ module.exports = {
       testPathIgnorePatterns: ["design-tokens"],
       testMatch: [
         "<rootDir>/packages/**/*.{ts|tsx}",
-        "tslint.json",
+        "eslint.json",
         "tsconfig.json",
+        "tsconfig.lint.json",
         "tsconfig.dist.json"
```

Comment:
I'm not entirely sure why the lint and tsconfig files are listed here, but I've swapped out `tslint` for `eslint`, and added the new `tsconfig.lint.json` file as well in case they need to be here for a good reason. If anyone knows why these are here, I'd love to know!